homeassistant/components/cloud/__init__.py

```diff
@@ -81,6 +85,43 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+class CloudNotAvailable(HomeAssistantError):
+    """Raised when an action requires the cloud but it's not available."""
+
+
+@bind_hass
+@callback
+def async_is_logged_in(hass):
```

Comment:
Should this have the `async_` prefix? It doesn't appear to do anything async...

Reply:
It's a callback, meaning we want to run inside the event loop.  It's OK.